polar-core/tests/integration_tests.rs

```diff
@@ -597,11 +597,11 @@ fn test_not() -> TestResult {
 
     // Negate And with unbound variable.
     p.load_str("i(x,y) if not (y = 2 and x = 1);")?;
-    qvar(&mut p, "i(2,y)", "y", values![sym!("_y_44")]);
+    qvar(&mut p, "i(2,y)", "y", values![sym!("y")]);
```

Comment:
So a variable being bound to itself in the output now means it is unbound? I don't think the previous output (bound to a temporary) was better, but I could see this being a source of confusion.